Sources/ApolloCodegenLib/UntypedGraphQLRequestBodyCreator.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+
+class UntypedGraphQLRequestBodyCreator {
+  
+  /// A non-type-safe request creator to facilitate sending requests not using code generation.
+  ///
+  /// Note: This is only public for use in `ApolloCodegenLib`, we do **not** recommend or support direct use of this in client applications.
```

Comment:
I question whether we need this comment:
* the class is not public
* developers don't import `ApolloCodegenLib` into their projects and even if they did this class has `internal` access.

If there is concern about this why don't we move the class into `ApolloSchemaDownloader.swift` with `fileprivate` access?

Reply:
Yep I had to back this out from being in the codegen lib because of cross-compilation issues. This whole comment can go away.